packages/tools/pie-wrapper-react/custom-elements.json

```diff
@@ -149,6 +149,14 @@
                     "description": "",
                     "name": "PieModal",
                     "members": [
+                        {
+                            "kind": "field",
+                            "name": "isDismissible",
```

Comment:
annoyingly I think you need to re-build the component and push the newer version of this file